Makefile

```diff
@@ -56,15 +62,16 @@ all: build test
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildTag "$(shell git describe --dirty)"
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildTime "$(shell date -u '+%Y/%m/%d %H:%M:%S')"
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildDeps "$(shell GOPATH=${GOPATH} build/depvers.sh)"
+build: TAGS += release
```

Comment:
Do we need to modify installsuffix when adding the release tag? Otherwise we could get a non-release build of a subpackage cached in `pkg/`, thus defeating the check we're trying to perform with this tag. (it's unfortunate though that a release build would recompile the world for such a small difference)